internal/infra-bionemo/src/infra_bionemo/license_check.py

```diff
@@ -44,8 +44,10 @@
     "main",
 )
 
-LICENSE_HEADER: str = """
-# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
One thing - should we update our `./license_header` to use `2024-2025 NVIDIA CORPORATION`? 
Our [current pre-commit](https://github.com/NVIDIA/bionemo-framework/blob/main/.pre-commit-config.yaml#L30) logic ignores this, it uses `./license_header` which has no 2025 mention, so this line was never being validated. I can update this and that file to both have it if we want it.